nodejs/week1/meal_sharing_website/src/backend/routes/meals.js

```diff
@@ -0,0 +1,35 @@
+//Respond with the json for all the meals.
+//For each meal, if there are reviews matching it's meal ID, add these reviews to each meal in the form of an array. 
+//For meals that do not have any reviews, the "reviews" property will be an empty array.
+
+const express = require("express");
+const router = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.